2.0/s2i/bin/run

```diff
@@ -10,6 +10,9 @@ set -e
 export RACK_ENV=${RACK_ENV:-"production"}
 
 if is_puma_installed; then
+  export_vars=$(cgroup-limits.py)
+  export $export_vars
```

Comment:
If you want it to be on one line, I can do `export_vars=$(cgroup-limits.py) ; export $export_vars`